### ISSUE TITLE
lokke on top of Guix (System) API

### ISSUE THREAD

**Issue by jgarte** [lokke-org/lokke#12]
Hi has anyone tested using lokke with Guix (System) APIs?

**Comment by jgarte** [comment]
@rlbdv It would be nice to be able to do

```lisp
lokke@lokke.user> gnu.packages.suckless/dwm
$1 = #<package dwm@6.2 gnu/packages/suckless.scm:215 7f2963565420>
```

**Comment by rlbdv** [comment]
@jgarte Now that I look back at this, I might have misinterpreted your question?  i.e. your example should already work with the `guile.` prefix, but perhaps you were hoping to avoid the prefix?

**Comment by jgarte** [comment]
Hi, yes I'd like to avoid the prefix. What do you suggest for that? defining aliases?

**Comment by jgarte** [comment]
I can just import qualify the guix stuff like here? https://github.com/lokke-org/lokke/blob/main/test/clojure-concurrency#L11

**Comment by rlbdv** [comment]
Definitely, `:refer` is the conventional way to handle that.  For what it's worth, I think I often tend to follow https://stuartsierra.com/2016/clojure-how-to-ns.html and https://github.com/bbatsov/clojure-style-guide

You can also use a shorter prefix if you prefer (as compared to a longer `:refer` list) via `:as something`.
